host_core/lib/host_core/actors/actor_rpc_server.ex

```diff
@@ -11,29 +12,17 @@ defmodule HostCore.Actors.ActorRpcServer do
       ) do
     pk = topic |> String.split(".") |> Enum.at(-1)
 
-    # NOTE - dispatch doesn't invoke the handler if no registry entries exist for the given key
+    case Registry.lookup(Registry.ActorRegistry, pk) do
+      [] ->
+        {:error, "Actor #{pk} is not running on this host. RPC call skipped."}
```

Comment:
Interesting edge case here. The scenario that causes this, a host that's listening on an RPC topic for an actor that's no longer running, is puzzling.

Perhaps here we could have the ActorRpcServer terminate that specific subscription? That may be a bad pattern, but if this message is received and there's no actor to handle it we probably shouldn't allow more messages to come to this host

Reply:
The RPC server will terminate the subscription when the actor is all out of instances. This is just to protect against ultra-low-chance race conditions where the actor process has died before the subscription has finished cleaning up.

Reply:
It's one of those things that should never realistically happen, but because code-wise it's physically possible, I still wanted to account for it to keep another process from crashing.